<?xml version="1.0" encoding="UTF-8"?>
<!--
    $Author: ronghui.xu  
    $Date: 2012-2-6 下午03:56:28  
    $Revision: 1.0  
    $Purpose: 
-->
<bm:model xmlns:e="aurora.service.exception" xmlns:o="aurora.database.local.oracle" xmlns:bm="http://www.aurora-framework.org/schema/bm" xmlns:f="aurora.database.features" alias="t1" baseTable="ord_sys_cus_bank_inf" defaultOrderBy="bank_branch_code">
    <bm:fields>
        <bm:field name="information_id" databaseType="NUMBER" datatype="java.lang.Long" physicalName="INFORMATION_ID" prompt="ord_sys_cus_bank_inf.INFORMATION_ID"/>
        <bm:field name="customer_id" databaseType="NUMBER" datatype="java.lang.Long" physicalName="CUSTOMER_ID" prompt="ord_sys_cus_bank_inf.CUSTOMER_ID"/>
        <bm:field name="bank_branch_code" databaseType="VARCHAR2" datatype="java.lang.String" physicalName="BANK_BRANCH_CODE" prompt="ord_sys_cus_bank_inf.BANK_BRANCH_CODE"/>
        <bm:field name="bank_account_code" databaseType="VARCHAR2" datatype="java.lang.String" physicalName="BANK_ACCOUNT_CODE" prompt="ord_sys_cus_bank_inf.BANK_ACCOUNT_CODE"/>
    </bm:fields>
    <bm:features>
        <o:sequence-pk/>
        <f:standard-who/>
    </bm:features>
    <bm:primary-key>
        <bm:pk-field name="information_id"/>
    </bm:primary-key>
    <bm:data-filters>
        <bm:data-filter enforceOperations="query" expression="t1.customer_id=${@customer_id}"/>
    </bm:data-filters>
    <e:exception-descriptor-config>
        <e:exception-descriptor exception="java.sql.SQLException" handleClass="aurora.database.SQLExceptionDescriptor">
            <e:error-message code="1" message="FND_SYSTEM.BANK_BRANCH_CODE_DISTINCT"/>
        </e:exception-descriptor>
    </e:exception-descriptor-config>
</bm:model>
